### DISCUSSION TITLE
Is WSDL generation supported?

### DISCUSSION THREAD

**Discussion by mohdali** [CoreWCF/CoreWCF#310] [General]
Hi, I just sww the announcement about the GA release and wonder if it will be possible to use this library right now to generate WSDL from contracts as we used to do in .NET Framework.

I understand it is not yet production ready but just wondering if this part is available to use now.

**Comment by birojnayak** [comment]
Not yet.. but this is one of our highest priority item as listed here https://github.com/CoreWCF/CoreWCF/issues/234#issuecomment-703430409 . So in the next release it should be available. @mconnew any more thought..

**Reply by mohdali** [reply]
Thanks, looking forward to it. Will keep an eye on the next releases.
